test/UtilTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+use LightStepBase\Client\Util;
+
+class UtilTest extends BaseLightStepTest {
+
+    public function testHexToDec() {
+        $util = new Util();
+        for ($x = 0; $x <= 100; $x++) {
+            $uuidStr = $util->_generateUUIDString();
```

Comment:
Out of scope: it would be nice if we had the ability to set a specific seed for our RNG so that this could be deterministic.